nx-dev/data-access-documents/src/lib/blog.api.ts

```diff
@@ -20,56 +20,33 @@ export class BlogApi {
     }
   }
 
-  async getBlogs(): Promise<BlogPostDataEntry[]> {
-    const files: string[] = await readdir(this.options.blogRoot);
-    const authors = JSON.parse(
-      readFileSync(join(this.options.blogRoot, 'authors.json'), 'utf8')
+  async getPodcastBlogs(): Promise<BlogPostDataEntry[]> {
+    return sortPodcasts(
+      await this.getBlogs((post) =>
+        post.tags.map((t) => t.toLowerCase()).includes('podcast')
+      )
     );
-    const allPosts: BlogPostDataEntry[] = [];
+  }
```

Comment:
Is there a reason Podcasts are using the "blog" api? Can we have the logic for Podcasts in a distinct api?

Reply:
Maybe we can rename it, but a podcast is basically a blog post with at the top having the voice/video included, some short description and potentially a transcript. That way when created it also appears as a blog post on the /blog (to get some more promotion automatically) and the handling is exactly the same.

Basically to simplify things in terms of data handling and authoring.

Reply:
I added a different API essentially it is the same but if we want to expand or update podcast APIs we can do it inside it's own concern